site/en/install/pip.md

```diff
@@ -51,7 +51,7 @@ step-by-step instructions.
     supported GPU on native-Windows.
     Starting with TensorFlow `2.11`, you will need to install
     [TensorFlow in WSL2](https://tensorflow.org/install/pip#windows-wsl2),
-    or install `tensorflow-cpu` and, optionally, try the
+    or install `tensorflow` or `tensorflow-cpu` and, optionally, try the
```

Comment:
```suggestion
    or install `tensorflow-cpu` and, optionally, try the
```

It looks like there's a TensorFlow 2.12 package for windows here:

https://pypi.org/project/tensorflow/#files

But it's only 2kB -> broken.   I think we should fix that.  What happens right now if you "pip install tensorflow" on windows?

Reply:
I think we don't want a `tensorflow` here. I'll patch.

Reply:
Hey Mark, TensorFlow on Windows is released by my Team at Intel. So 2kB is fine. It internally downloads tensorflow_intel https://pypi.org/project/tensorflow-intel/#files